source/code/plugins/in_npmd_server.rb

```diff
@@ -334,6 +343,34 @@ def check_and_update_binaries
             File.unlink(_x64BinPath) if File.exist?(_x64BinPath)
         end
 
+        def check_agent_capability
+            _deleteVersionFiles = false
+
+            if !File.exist?(@location_agent_binary)
+                _deleteVersionFiles = true
+            else
+                _getCapResult = `getcap #{@location_agent_binary}`
```

Comment:
Have you verified that the `getcap` program exists on [all distros/versions we support](https://github.com/Microsoft/OMS-Agent-for-Linux/tree/master#supported-linux-operating-systems)?